support/src/main/resources/org/jenkinsci/plugins/workflow/support/steps/build/BuildTriggerStep/help-job.html

```diff
@@ -0,0 +1,7 @@
+<div>
+    Name of a downstream job to build.
+    May be another Workflow job, but more commonly a freestyle or other project.
+    Use a simple name if the job is in the same folder as this upstream workflow job;
+    otherwise can use relative paths like <code>../sister-folder/downstream</code>
+    or absolute paths like <code>/top-level-folder/nested-folder/downstream</code>.
```

Comment:
Yes…but I do not want to advertise that, because it is only for compatibility with historical settings, which does not apply here.